worlds/ror2/Options.py

```diff
@@ -123,14 +123,14 @@ class ItemPoolPresetToggle(DefaultOnToggle):
     display_name = "Item Weight Presets"
```

Comment:
Maybe you could change the display name of this option as well to help make it clearer?